packages/components/src/styles/Button.module.css

```diff
@@ -14,12 +14,16 @@
 }
 
 .base:focus-visible {
```

Comment:
Yeah for pressed/active state the browser throws on `:focus-visible` with an outline for some reason. To prevent that I added this so only [data-focus-visible] shows an outline.

Reply:
oh i see, you're turning the outline. makes sense.